src/orchard/eldoc.clj

```diff
@@ -3,6 +3,10 @@
   in editors."
   {:author "Bozhidar Batsov"})
 
+(def ^:private type-abbrevs
```

Comment:
If we go this route this should be something optional (and ideally something that clients can modify).

Reply:
Yeah, I agree - it should probably be a client-side processing step, eg. eldoc could do a search-and-replace via regex.